articles/zcash-protocol/chapter_1.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: Introduction
+Status: Done
+Level: "1"
+---
+# Chapter 1: Introduction
+---
+Zcash, an evolution of the Zerocash protocol, bridges the gap between transparent Bitcoin transactions and shielded payments enabled by [zk-SNARKs](../../terms/zkSNARK.md). 
+
+**Concepts and Abstractions:**
+- **Zero-knowledge proofs:** The magic behind Zcash's shielded transactions. These cryptographic tools allow users to prove the validity of transactions without revealing their details, enabling private spending.
+- **Shielded addresses:** These special addresses mask spending and receiving addresses, further enhancing privacy.
+- **Sapling:** The upgraded protocol was introduced in 2018, offering smaller and faster transactions compared to its predecessor, Sprout.
+
+**Protocol Deep Dive:**
+- **Abstract protocol:** Defines the ideal components of Zcash, including functions for generating proofs, verifying their validity, and managing shielded addresses.
+- **Concrete protocol:** Explains how these ideal functions are implemented using specific cryptographic algorithms and data structures.
+- **Network upgrades:** Outlines the process for introducing new features and functionality to Zcash over time, ensuring its ongoing development and evolution.
+
+**Beyond Transparency:**
+- **Consensus changes from Bitcoin:** Zcash deviates from Bitcoin's Proof-of-Work consensus mechanism to accommodate shielded transactions efficiently.
+- **Differences from Zerocash:** The document analyzes the key modifications Zcash made to the original Zerocash protocol, improving security, performance, and usability.
+
+**Technical Appendix:**
+- **Circuit design:** Provides detailed insights into how Sapling circuits, the mathematical core of zk-SNARKs, are designed and implemented.
+- **Batching optimizations:** Explains techniques for verifying multiple signatures and proofs simultaneously, improving efficiency and scalability.
+
+### 1.1 High-level Overview
+---
+- This overview contains the ideas behind the protocol, but it is imprecise in some aspects and is not a part of the normative protocol specification.
+- Also, this overview applies to Sprout, Sapling, and Orchard.
+Zcash uses different value pools to manage transparent and shielded transactions. The transparent pool works similarly to Bitcoin, while shielded pools (Sprout, Sapling, Orchard) leverage notes for encrypted transactions.
+- These notes contain amounts, shielded addresses, spending keys, and commitments for verification. To prevent double-spending, nullifiers linked to spending keys invalidate used notes. 
```

Comment:
```suggestion
- Shielded transactions contain amounts, shielded addresses, spending keys, and commitments for verification. To prevent double-spending, nullifiers linked to spending keys to invalidate used transactions. 
```